aep/general/0300/aep.md.j2

```diff
@@ -0,0 +1,98 @@
+# AEP Editions
+
+This AEP describes how the AEP specification is versioned.
+
+## Edition based versioning
+
+AEPs will be published in editions, similar but not identical to specifications
+such as those for the C++ or Rust programming language:
+
+- The specification will not change for a non-preview edition after it is
+  published.
+- The naming of the edition will be based on the year in which it is published
+  (e.g. aep-2025).
+- Upon publishing a non-preview edition, the current prevision will be removed,
+  and the next preview edition will be created (e.g. aep-2027-preview). This
+  preview edition will accumulate changes until the next edition is published.
+- AEP editions **MAY** have patch version updates which adhere to semantic
+  versioning (e.g. aep-2025.25).
+  - The patch version will only contain fixes to typos and clarification on
+    existing guidance.
```

Comment:
```suggestion
  - The patch version will only contain non-breaking changes such as fixes to typos in, and clarification on,
    existing guidance.
```

Reply:
I think this leaves some wiggle room as to what a "non-breaking change" is, and as we discussed, basically any change to the spec is a breaking change.

I'm generally against introducing breaking / non-breaking distinction, and prefer just enumerating these exceptions here. Would that be ok?

Reply:
```suggestion
  - The patch version will only contain fixes to typos in, and clarification on,
    existing guidance.
```

Here's a suggestion compatible with that, in the interim.

But the challenge is that if we think of another kind of non-breaking change, and haven't enumerated it here, it could be a breaking change to add it to this list.